src/App.js

```diff
@@ -1,10 +1,22 @@
+import { Route, Routes, useLocation } from "react-router-dom";
 import "./App.css";
-import { LandingPage } from "./pages/LandingPage/LandingPage";
+
+import { LandingPage, HomePage, Archive, Labels, Trash } from "./pages";
 
 function App() {
+  const { pathname } = useLocation();
+  console.log(pathname);
   return (
     <>
-      <LandingPage />
+      {/* {pathname !== "/" && <Navbar />} */}
```

Comment:
Remove commented code.

Reply:
i have kept it for  future use to refactor my code in future